docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -71,66 +75,53 @@ Then you will get the binary distribution under **distribution/target**, in whic
 | sbin                    | Shell files folder, contains start/stop/remove shell of ConfigNode and DataNode, cli shell |
 | tools                   | System tools                                                                               |
 
-## Start the Cluster
-
-Users could start a cluster which contains multiple ConfigNode and DataNode.
-A cluster need at least one ConfigNode and no less than the number of data/schema_replication_factor DataNodes.
-
-The total process are three steps:
+# 5. Cluster Installation and Configuration
 
-* Start the first ConfigNode
-* Add ConfigNode (Optional)
-* Add DataNode
+## 5.1 Cluster Installation
 
-### Start the first ConfigNode
+Deploy apache-iotdb-1.0.0-SNAPSHOT-all-bin to the specified directory on the server or VM 
+to complete cluster installation on a server or VM.
+Each apache-iotdb-1.0.0-SNAPSHOT-all-bin can start one ConfigNode and one DataNode.
 
-Please set the important parameters in conf/iotdb-confignode.properties and conf/iotdb-common.properties:
+## 5.2 Cluster Configuration
 
```

Comment:
We need to modify the configurations on each server.
Therefore, login each server and switch the working directory to apache-iotdb-1.0.0-SNAPSHOT-all-bin.
The configruation files are stored in the `./conf` directory.

For all ConfigNode servers, we need to modify the common configuration (see Chap 5.2.1) and ConfigNode configuration (see Chap 5.2.2).

For all DataNode servers, we need to modify the common configuration (see Chap 5.2.1) and DataNode configuration (see Chap 5.2.3).

Reply:
Fixed~